vscode/src/commands.ts

```diff
@@ -51,12 +51,37 @@ const commandsMap: (state: ExtensionState) => {
       if (!(await analyzerClient.canAnalyzeInteractive())) {
         return;
       }
-
       try {
         await analyzerClient.start();
         await analyzerClient.initialize();
       } catch (e) {
-        console.error("Could not start the analyzer", e);
+        console.error("Could not start the server", e);
+      }
+    },
+    "konveyor.stopServer": async () => {
```

Comment:
Do we want to tie this into the stop server button in the webview or create a GH issue to track that?

Reply:
I figure any updates to the web view would be better in another pr